test/npu-xrt/add_one_two/Makefile

```diff
@@ -0,0 +1,40 @@
+include ../makefile-common
+
+targetname = addOneObjfifo
+
+all: build2/final.xclbin addOneObjfifo.exe
+
+build1/first.xclbin: aie1.mlir
+	mkdir -p ${@D}
+	cd ${@D} && aiecc.py \
+				--xclbin-kernel-name=ADDONE --xclbin-kernel-id=0x901 --xclbin-instance-name=ADDONEINST\
```

Comment:
I'm skeptical of all the xclbin specific command line options, but I guess this is good enough for now.

Reply:
removed